Automattic/wp-calypso
# Me: Delete legacy ApplicationPasswords store

This PR removes the legacy `ApplicationPasswords` event emitter store as it's no longer used. This is the last PR of #22912. 

Fixes #22912.

To test:
* Checkout this branch
* Verify there are no usages of `lib/application-passwords-data` left.
